mlkem/indcpa.c

```diff
@@ -193,6 +199,32 @@ __contract__(
   ensures(array_bound(data, 0, MLKEM_N, 0, MLKEM_Q))) { ((void)data); }
 #endif /* !MLK_USE_NATIVE_NTT_CUSTOM_ORDER */
 
+static void mlk_transpose_matrix(mlk_polymat a)
```

Comment:
Can you add this to the component benchmarks so we get a sense of the performance cost?